22_pipeslab/pipeslab.txt

```diff
@@ -3,19 +3,42 @@ For each of the following, type a single command line that performs:
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
 
+  tocode@tocode-VirtualBox:~$ ls -alF | grep tocode | cut -c4 | grep x | wc | cut -d' ' -f6
+  20
+  grep x is not must
+  
   2. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner, group and others.
   Answer: 
-
+ 
+  ls -alF | grep tocode | grep x..x..x | wc | cut -d' ' -f6
+  
+  
   3. Print the number of lines from "ls -l" which represent paths that were modified today (no matter which day we run the command)
   Answer: 
 
+  ls -alF | grep tocode | grep Jan..2 | wc | cut -d' ' -f7
+  
   4. Modify (1) and (2) to count files only (no directories)
   Answer:
+  
+	ls -alF | grep tocode | grep -v / | cut -c4 | grep x | wc | cut -d' ' -f7
+	
+	ls -alF | grep tocode | grep -v / | grep x..x..x | wc | cut -d' ' -f7
 
+	
   5. The command /sbin/ifconfig shows information regarding the machine's network interfaces. Use the command to print only the IP address of the machine. Remember not to show the special loopback address 127.0.0.1
+	
+	date:
+	ls -alF | grep "$(date '+%b %e')"
```

Comment:
I think you have issue with the order in the web (in hebrew) and in the files...